rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -39,14 +42,21 @@ final case class Cache[C, P, A, K](
 
 private class InMemHotStore[F[_]: Sync, C, P, A, K](
     implicit S: Cell[F, Cache[C, P, A, K]],
-    HR: HistoryReader[F, C, P, A, K]
+    HR: HistoryReader[F, C, P, A, K],
+    cp: Codec[P],
+    ck: Codec[K]
 ) extends HotStore[F, C, P, A, K] {
 
   def getContinuations(channels: Seq[C]): F[Seq[WaitingContinuation[P, K]]] =
     for {
       cached <- internalGetContinuations(channels)
       state  <- S.read
-    } yield (state.installedContinuations.get(channels) ++: cached)
+      result = state.installedContinuations.get(channels) ++: cached
+      res <- {
+        implicit val codec = fromCodec(implicitly[Codec[Seq[WaitingContinuation[P, K]]]])
+        roundTrip[F, Seq[WaitingContinuation[P, K]]](result)
+      }
+    } yield res
```

Comment:
My understanding is that this change fixes the ignored test. Can you explain how this change affects this behaviour?

Reply:
It's described in detail in https://rchain.atlassian.net/browse/RCHAIN-3257

Reply:
I see. The PR is now pointing to this Jira: https://rchain.atlassian.net/browse/RCHAIN-3288 which doesn't contain that information. Maybe you should link the two jiras to make it obvious for the future generations